_data/navigation.yml

```diff
@@ -420,8 +422,6 @@ reference:
       url: /reference/pipeline/expressions/
     - title: Pipeline Stages
       url: /reference/pipeline/stages/
-  - title: Deployment Infrastructure
```

Comment:
Removed?

Reply:
Not really, @ttomsu. It never existed. It was something I started in another branch, but which I accidentally leaked into this one :/